isobus/include/isobus/isobus/isobus_virtual_terminal_working_set_base.hpp

```diff
@@ -92,6 +92,7 @@ namespace isobus
 
 		std::mutex managedWorkingSetMutex; ///< A mutex to protect the interface of the managed working set
 		VTColourTable workingSetColourTable; ///< This working set's colour table
+		std::uint32_t iopSize = 0, transferredIopSize = 0;
```

Comment:
Nit (non-blocking): Prefer two different lines for consistency with our other repos.